snuba/split.py

```diff
@@ -114,40 +112,33 @@ def time_split(dataset, body, *args, **kwargs):
 
         return overall_result, status
 
-    def col_split(dataset, body, *args, **kwargs):
+    def col_split(dataset, query, options, *args, **kwargs):
         """
         Split query in 2 steps if a large number of columns is being selected.
             - First query only selects event_id and project_id.
             - Second query selects all fields for only those events.
             - Shrink the date range.
         """
-        minimal_query = {**body, 'selected_columns': MIN_COLS}
-
-        result, status = query_func(dataset, minimal_query, *args, **kwargs)
+        minimal_query = {**query, 'selected_columns': MIN_COLS}
+        result, status = query_func(dataset, minimal_query, options, *args, **kwargs)
 
         # If something failed, just return
         if status != 200:
             return result, status
 
-        conditions = body.get('conditions', [])
-
         if result['data']:
-            project_ids = list(set([event['project_id'] for event in result['data']]))
-            body['project_id'] = project_ids
```

Comment:
This should have been `project`. I missed this during the initial review (I may have even suggested the wrong key. 😬 I feel like we can't get to type checks soon enough.)